source/usage-examples/find.txt

```diff
@@ -0,0 +1,143 @@
+=======================
+Find Multiple Documents
+=======================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+`collection.find() <https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#find>`_
+lets you query for multiple documents in MongoDB. You can pass ``find``
+an (optional) query document and an options object. Use the options
```

Comment:
Agreed, especially since the purpose of this is sort of a jumping-off point for new users/users who have forgotten the concept but may have learned it before.